apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/sharing-modal/suggested-tags.tsx

```diff
@@ -0,0 +1,116 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button, FormTokenField } from '@wordpress/components';
+import { TokenItem } from '@wordpress/components/build-types/form-token-field/types';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { store as noticesStore } from '@wordpress/notices';
+import { useI18n } from '@wordpress/react-i18n';
+import * as React from 'react';
+import useAddTagsToPost from './use-add-tags-to-post';
+
+type PostMeta = {
+	reader_suggested_tags: string;
+};
+
+type CoreEditorPlaceholder = {
+	getCurrentPost: ( ...args: unknown[] ) => {
+		id: number;
+		meta: PostMeta;
+	};
+};
+
+type SuggestedTagsEventProps = {
+	number_of_original_suggested_tags: number;
+	number_of_selected_tags: number;
+	number_of_suggested_tags_selected: number;
+	number_of_added_tags: number;
+};
+
+type SuggestedTagsProps = {
+	setShouldShowSuggestedTags: ( shouldShow: boolean ) => void;
+};
+
+function SuggestedTags( props: SuggestedTagsProps ) {
+	const { __, _n } = useI18n();
+	const { id: postId, meta: postMeta } = useSelect(
+		( select ) => ( select( 'core/editor' ) as CoreEditorPlaceholder ).getCurrentPost(),
+		[]
+	);
+	const { createNotice } = useDispatch( noticesStore );
+	const origSuggestedTags = postMeta?.reader_suggested_tags
```

Comment:
Should we `setShouldShowSelectedTags( false )` if `origSuggestedTags` has 0 length?

Right now we start it as true, and set it as false once we click to add the tags. But in the case that there are no tags to suggest it seems like it would remain open with no suggestions?

Reply:
Good idea! It should be false if no suggested tags are found.